.travis.yml

```diff
@@ -37,17 +40,54 @@ env:
 matrix:
   fast_finish: true
   include:
+# PHP versions compatibility (latest WordPress and most used WooCommerce versions)
```

Comment:
How is the most used version identified? 
Can we automate this check or assign result to internal env var in earlier phases to simplify management (repo level env var might work here as well, we could update it without PR)? Idea:
```yml
...
# in the matrix
  php: 7.0
  env: WP_VERSION=latest WC_VERSION=$WC_MOST_USED_VERSION

...

before_install:
  - WC_MOST_USED_VERSION=get_mostu_used_vs
...
```

Reply:
We can introduce the variable, no problems. `WC_MOST_USED_VERSION='fixed-version-here'` will be fixed as the version identified manually.

Reply:
Sounds good. Same applies to min/max versions. It's nice to have them defined in the code and see with git history who/when updated them :)